gnome-selected-20170316.json

```diff
@@ -0,0 +1,16 @@
+{
+  "tags": [
+    "ENV-DISTRI-opensuse",
+    "gnome-selected"
+  ],
+  "area": [
+    {
+      "width": 162,
+      "type": "match",
+      "xpos": 358,
+      "ypos": 373,
+      "height": 12
+    }
+  ],
+  "properties": []
+}
```

Comment:
technically the needle is fine but the design of the installer page is not very convincing. This was also mentioned to yast team in mailing list threads, irc channel, pull requests. I don't know how to continue here but I am objecting the acceptance of the state as is.

@nilxam not your fault